### PR TITLE
start R6 document so that we have a place to add items as development…

### PR DESCRIPTION
**Pull request by fkr**
… progresses

### CODE REVIEW

File: Release6.md [hunk 1]
```diff
@@ -55,6 +75,9 @@ Overall with Release 6 it becomes apparent that SCS is efficient to operate.
 
 ## List of known issues & restrictions in R6
 
+### KaaS
+* Some features of KaaS v1 are not available yet in KaaS v2 because they are WIP in upstream CAPO
```
Comment (fkr) [inline Release6.md]: Would it make sense to list these features here?

**Comment by markus-hentsch** [comment]
I added the domain related work as preview sections.

**Comment by joshmue** [comment]
Added a section about the central API.

File: Release6.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+mapping of users, groups and roles from OpenID-Connect token claims to the OpenStack Keystone access management.
+
+
+### Upcoming: automated pentesting security pipeline
+
+In the beginning of 2024 we started with the conception and implementation of an automated security pipeline. This pipeline will serve as a security scanning tool that is either triggered by deployments or due to manual activation in your infrastructure. The pipeline will contain six tools: Naabu, httpx, Nuclei, Greenbone (Community Edition), ZAP and DefectDojo. While the first three are meant to discover network-related issues like open ports and attack vectors, GCE and ZAP will deliver deeper security inspections of hosts in your SCS environment. Reports are collected in DefectDojo, allowing an overview over security-related changes and seeing the current state. The pipeline will be available as a platform tool in R7.
```
Comment (fkr) [inline Release6.md]: I'm not really happy with this. These are technical release notes and we should focus on the stuff we are releasing. This seems good content for an outlook blogpost imho.

Comment (garloff) [reply Release6.md]: Well, maybe we shorten this and attach it to the section that talks about the pentesting that already happened and that improved R6 security?

**Review by garloff — APPROVED** [review]
I think this is in a reasonable shape now.

**Pull request merged**